app/pom.xml

```diff
@@ -14,7 +14,7 @@
         <main.class.name>com.oracle.javafx.scenebuilder.app.SceneBuilderApp</main.class.name>
         <!-- For about.properties file -->
         <buildDate>${maven.build.timestamp}</buildDate>
-        <javaVersion>${java.version}, ${java.runtime.name}</javaVersion>
+        <javaVersion>${java.version.app}, ${java.runtime.name}</javaVersion>
```

Comment:
Isn't this property only used to show details in the application and doesn't have any effect on how the `app` project is built via Maven.

If you want `app` to be built using Java 17, `<java.version>17</java.version>` needs to be added to `app/pom.xml` properties section.

Reply:
This is indeed a mistake. There is now a property `<java.version>${java.version.app}</java.version>` and it runs.